### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by descorp**
# Open PR

## Changes

* fix link to Configuration docs
* add links to Adyen Docs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,7 +108,7 @@ For general understanding of how prebuilt UI components of Adyen work you can fo
 
 ### Configuration
 
-To read more about other configuration, see [Full list](docs\Configuration.md).
+To read more about other configuration, see [full list](configuration).
```
Comment (mattlies) [inline README.md]: Suggestion: "see the [full list](configuration)"

File: README.md [hunk 2]
```diff
@@ -178,16 +178,18 @@ const handleSubmit = (payload, nativeComponent) => {
 
 ## Documentation
 
-- [Configuration](/docs/Configuration.md)
+- [Configuration][configuration]
 - [Error codes](/docs/Error%20codes.md)
-
-> :construction: ** Adyen Docs documentation in progress**
+- [Drop-in Documentation][adyen-docs-dropin]
```
Comment (mattlies) [inline README.md]: Suggestion: lowercase "documentation"

File: docs/Configuration.md [hunk 3]
```diff
@@ -1,13 +1,15 @@
...
-
-### Root configurations
+## Root configurations
 * **environment** - Use `test`. Project is currently in **BETA** stage; to do penny-test, change the value to one of our [live environments](https://docs.adyen.com/online-payments/drop-in-web#testing-your-integration).
 * **clientKey** - A public key linked to your API credential, used for [client-side authentication](https://docs.adyen.com/development-resources/client-side-authentication).
-* **amount** - Amount to be displayed on the Pay Button. It expects an object with the minor units value and currency properties. For example, { value: 1000, currency: 'USD' }. For BIN or card verification requests, set amount to 0 (zero).
-* **countryCode** - The shopper's country code in ISO 3166-1 alpha-2 format. Example: NL or DE.
-* **shopperLocale** - 🚧 Work in progress. In current version this property only localises payment methods names. Default OS's locale is used for localisation. You can override  
+* **amount** - Amount to be displayed on the Pay Button. It expects an object with the minor units value and currency properties. For example, { value: 1000, currency: 'USD' }. For card pre-authorisation set amount to 0 (zero).
```
Comment (mattlies) [inline docs/Configuration.md]: Style guide and consistency with Docs suggestion: Use `monospace` for parameters and **strong** for the values. `amount`: ... For example, `{`  `value`: **1000**, `currency`: **USD**  `}`

File: docs/Configuration.md [hunk 4]
```diff
@@ -1,13 +1,15 @@
...
+* **countryCode** - The shopper's country code in ISO 3166-1 alpha-2 format. Example: NL or DE. 
+* **shopperLocale** - 🚧 Work in progress. In current version this property only localises payment methods names. Default OS's locale is used for localisation.
+* **returnUrl** - For iOS, this is the URL to your app, where the shopper should return, after a redirection. Maximum of 1024 characters. For more information on setting a custom URL scheme for your app, read the [Apple Developer documentation](https://developer.apple.com/documentation/uikit/inter-process_communication/allowing_apps_and_websites_to_link_to_your_content/defining_a_custom_url_scheme_for_your_app).
+For Android, this value is automatically overridden by AdyenCheckout.
```
Comment (mattlies) [inline docs/Configuration.md]: Style guide and consistency with Docs suggestion: also `monospace` for class/file names: `AdyenCheckout`

File: README.md (hunk 1 above)
Comment (descorp) [reply README.md]: Fixed [5aecb70](https://github.com/Adyen/adyen-react-native/pull/146/commits/5aecb7059029f427ac1ae67cfc209baf38c8cedf)

File: README.md (hunk 2 above)
Comment (descorp) [reply README.md]: Fixed [5aecb70](https://github.com/Adyen/adyen-react-native/pull/146/commits/5aecb7059029f427ac1ae67cfc209baf38c8cedf)

File: docs/Configuration.md (hunk 4 above)
Comment (descorp) [reply docs/Configuration.md]: Fixed [d53a296](https://github.com/Adyen/adyen-react-native/pull/146/commits/d53a296032f048de0fec4cc5aeeed235e3755a7e)

File: docs/Configuration.md (hunk 3 above)
Comment (descorp) [reply docs/Configuration.md]: Fixed [d53a296](https://github.com/Adyen/adyen-react-native/pull/146/commits/d53a296032f048de0fec4cc5aeeed235e3755a7e).
In the case above - used  `{ value: 1000, currency: USD }`

**Review by camil-adyen — APPROVED** [review]

**Review by erenbesel — APPROVED** [review]

**Review by neelSharma12 — APPROVED** [review]

**Pull request merged**